portality/forms/application_forms.py

```diff
@@ -2719,9 +2719,9 @@ def __call__(self, field, **kwargs):
         html = ['<%s %s>' % (self.html_tag, html_params(**kwargs))]
         for subfield in field:
             if self.prefix_label:
-                html.append('<li>%s %s' % (subfield.label, subfield(**kwargs)))
+                html.append('<li tabindex=0>%s %s' % (subfield.label, subfield(**kwargs)))
```

Comment:
@richard-jones I don't think this is the best place to put that but I cannot figure out what would better place be. IT does not feel like a new widget for formulaic either... What do you think?

Reply:
I think you'll have to talk me through this one, I'm not sure what bug is being fixed, and what the trade-offs here are.

Reply:
This line will fix https://github.com/DOAJ/doajPM/issues/2993